omero/conf.py

```diff
@@ -136,6 +136,11 @@
 # Regular expressions that match URIs that should not be checked when doing a linkcheck build
 linkcheck_ignore += [r'http://localhost:\d+/?', 'http://localhost/',
     'http://www.hibernate.org',
+    'http://www.jboss.org',
+    'https://code.google.com/archive/p/luke/',
+    'https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ',
+    'https://httpd.apache.org/docs/current/mod/mod_proxy.html',
+    'https://access.redhat.com/articles/3078',
```

Comment:
As discussed separately, some of the decisions of https://trello.com/c/ccx2knE7/528-check-whether-sphinx-can-be-upgraded-for-all-docs-jobs effectively forced us to increase the stringency of our CI link checking which now reports any failure rather than filtering the most important status codes (403, 404).

To allow us to make incremental steps, seeing the lists of ignored links extended in a reasonable fashion is completely fine as these links were probably filtered out from the report anyways. We can investigate whether this list can be reduced using options like increasing the linkcheck timeout once we have stabilized the documentation state.